src/include/daos/tests_lib.h

```diff
@@ -14,16 +14,17 @@
 #include <daos/object.h>
 #include <daos/credit.h>
 
-#define assert_rc_equal(rc, expected_rc)				\
-	do {								\
-		if ((rc) == (expected_rc))				\
-			break;						\
-		print_message("Failure assert_rc_equal %s:%d "		\
-			      "%s(%d) != %s(%d)\n", __FILE__, __LINE__, \
-			      d_errstr(rc), rc,				\
-			      d_errstr(expected_rc), expected_rc);	\
-		assert_string_equal(d_errstr(rc), d_errstr(expected_rc)); \
-		assert_int_equal(rc, expected_rc);			\
+#define assert_rc_equal(rc, expected_rc)                                                           \
+	do {                                                                                       \
+		int __rc = (rc);                                                                   \
```

Comment:
the primary change in the macro is (rc) only appears once.  The rest is clang_format